### DISCUSSION TITLE
Multiple segment timelines for different cuts, or other purpose like chapters, subtitles..

### DISCUSSION THREAD

**Discussion by devtweakr** [mifi/lossless-cut#2097] [Ideas]
Here are my suggestions for multiple segment timelines:

In tracks dialog, Timelines section

Additional Icon to add new timeline (asks for name dialog)

In 3 dots menu for timeline

1. Rename
2. Duplicate the timeline
3. Discard (confirmation dialog)

Additional icons after the keyframe icon:

1. Dropdown select for editing (like zoom dropdown, with eye icon before each timeline name to select which timelines are visible)
1. All timelines visible on/off (if off, only timelines that are marked visible in the dropdown are visible)

Interaction between visible timelines:

Drag and drop functionality to move segments between timelines, with optional ctrl-drag to copy segments between timelines. 

Export options for timelines:

1. Selection of Cut timeline dropdown (which timeline to cut / merge)
2. Selection of Chapters timeline (optional, dropdown to select which timeline are chapters, when export to format like MKV that supports chapters)
3. Selection of Subtitles timeline (optional, dropdown to select which timeline are subtitles, with simultaneous export of SRT file)

I hope this functionality is not too complicated to add :)

**Comment by mifi** [comment]
are you talking about #976?

**Reply by devtweakr** [reply]
No, i think the main purpose of this tool is just cut that doesn't require re-encoding, working with one file at the time.
I would just add (duplicate) timeline functionality, if needed to make different cuts of the same file, chapters, subtitles, etc..
Or just to keep one track with notes for example, for the purpose of marks and do the fine cut on another.

Does this make sense?

**Reply by mifi** [reply]
Ah ok. You basically want to have one or more `.llc` projects visible at the same time on the timeline, and also be able to interact between them (move segments back and forth). It's definitely not a trivial change, but I'll add it here: #483

**Reply by devtweakr** [reply]
yes this could work,

llc projects treated as tracks maybe, so you can manage them under tracks menu, name, selection main timeline (default), importing subtitles or other timestamp files which would appear as different timelines (tracks)
in main UI they would appear as multiple timelines (multiple rows) if selected a visible.
